.pre-commit-config.yaml

```diff
@@ -42,3 +54,35 @@ repos:
     -   id: check-xml
     -   id: check-merge-conflict
     -   id: check-case-conflict
+
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.9.0  # pick a git hash / tag to point to
+    hooks:
+    -   id: flake8
+
+-   repo: https://github.com/NanoSurface-Biomedical/pre-commit-pylint
+    rev: "5a76725fa664ca733db485598da5c8460cf3347f" # pick a git hash / tag to point to
+    hooks:
+    -   id: pylint-pass-with-displayed-warnings
+        name: pylint-src
+        args:
+        - --codes-to-allow=W0511
+        - src
+        - --rcfile=src/pylintrc
+        - --ignore-patterns=.*~.*
+        - --jobs 0
+        files: ('src/.*\.py$')
+        exclude: '.*\~.*'
+        verbose: true
+    -   id: pylint-pass-with-displayed-warnings
+        name: pylint-tests
+        args:
+        - --codes-to-allow=W0511
+        - tests
+        - --rcfile=tests/pylintrc
+        - --ignore-patterns=.*~.*
+        - --jobs 0
+        files: 'tests/.*\.py$'
+        exclude: ('.*\~.*')
+        verbose: true
+    -   id: python-check-no-print-statments
```

Comment:
add a newline

Reply:
implemented.  do you know if there's a pre-commit hook that automatically adds extra newlines to the end of files?

Reply:
I think ``black`` does this for the Python files, but it doesn't parse YAML